Ps 112 upgrading ha rhel8 21.10 to 22.10 (#2152)
centreon/centreon-documentation: versioned_docs/version-22.10/upgrade/centreon-ha/upgrade-from-21-10.md

### Perform a backup

Be sure that you have fully backed up your environment for the following servers:

- Central server
- Database server

### Update the RPM signing key

For security reasons, the keys used to sign Centreon RPMs are rotated regularly. The last change occurred on October 14, 2021.
When upgrading from an older version, you need to go through the [key rotation procedure](../../security/key-rotation.md#existing-installation), to remove the old key and install the new one.

## Upgrade process

To perform the upgrade:

> For the **active central node** and **active database node if needed** please [follow the official documentation](../../upgrade/upgrade-from-21-10.md) **until the "Post-upgrade actions" step included**.

> > For the **passive central node** and **passive database node if needed**, please [follow the official documentation](../../upgrade/upgrade-from-21-10.md) **until the "Update your customized Apache configuration" step included only. Do not proceed the "Finalizing the upgrade" step.**.

Reviewer: ```suggestion
> For the **passive central node** and **passive database node if needed**, please [follow the official documentation](../../upgrade/upgrade-from-21-10.md) **until the "Update your customized Apache configuration" step included only. Do not perform the "Finalizing the upgrade" step.**.
```

> > **WARNING:** the syntax of the following command depends on the Linux Distribution you are using.

Reviewer: ```suggestion
> **WARNING:** the syntax of the following command depends on the Linux distribution you are using.
```

> > You can find @CENTRAL_MASTER_NAME@ @CENTRAL_SLAVE_NAME@ @MARIADB_REPL_USER@ @MARIADB_REPL_USER@ variable in `/etc/centreon-ha/mysql-resources.sh`

Reviewer: ```suggestion
> You can find the @CENTRAL_MASTER_NAME@ @CENTRAL_SLAVE_NAME@ @MARIADB_REPL_USER@ @MARIADB_REPL_USER@ variables in `/etc/centreon-ha/mysql-resources.sh`
```

> > **WARNING:** the syntax of the following command depends on the Linux Distribution you are using.

Reviewer: ```suggestion
> **WARNING:** the syntax of the following command depends on the Linux distribution you are using.
```
